test/functional/rpc_scanblockfilters.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the scanblocks rpc call."""
```

Comment:
nit: What about:

```suggestion
"""Test the scanblocks RPC call."""
```
?

It looks like it is mostly written in this way in tests.

Reply:
Fixed in #23549